BackEnd/routes/members.py

```
@@ -1,15 +1,101 @@
+        return {"status": "success", "data": {"members": members, "total": len(members)}}
+    except Exception as e:
+        logger.error(f"Error getting all members: {str(e)}")
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=str(e))
+
+@router.put("/members/{member_id}")
+async def update_member_points(member_id: str, data: dict):
+    """Update member's total points"""
+    try:
+        try:
+            ObjectId(member_id)
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)